### PR TITLE
Move tests from times module to ttime.nim test file

### PR DESCRIPTION
**Pull request by jlp765**

### CODE REVIEW

**Comment by Araq** [comment]
Looks good to me. Ping @dom96

File: lib/pure/times.nim [hunk 1]
```diff
@@ -1306,13 +1204,3 @@ when isMainModule:
   assert toSeconds(t4L, initInterval(days=1)) == toSeconds(t4, initInterval(days=1))
   assert toSeconds(t4L, initInterval(months=1)) == toSeconds(t4, initInterval(months=1))
   assert toSeconds(t4L, initInterval(years=1)) == toSeconds(t4, initInterval(years=1))
```
Comment (dom96) [inline lib/pure/times.nim]: Please add a comment after this which says where the public functions are tested.

**Comment by dom96** [comment]
Any ideas why travis is failing?

**Comment by jlp765** [comment]
Travis failing:

3 tests fail (this is for everything, not just what I have committed)
```
tests/concepts/tmanual.nim
tests/misc/tlocals.nim
tests/types/tisopr.nim
```
On my local win7 PC, 
- running each individually works as expected (output matches what is expected).
- running via `koch test r tests\...\t....nim`  fails as follows:
 - `tisopr` passes (but Travis fails with `unhandled exception: false`)
 - `tmanual` fails with `type mismatch: got (seq[int])`
 - `tlocals` fails with `internal error: (filename: ast.nim, line: 1561)`

`tmanual` code doesn't return a seq[int], but echo's individual values

Line 1561 in ast.nim is the internalAssert in the following:
```
proc skipGenericOwner*(s: PSym): PSym =
  internalAssert s.kind in skProcKinds
  ## Generic instantiations are owned by their originating generic
  ## symbol. This proc skips such owners and goes straight to the owner
  ## of the generic itself (the module or the enclosing proc).
  result = if sfFromGeneric in s.flags: s.owner.owner
           else: s.owner
```

So in summary:
- tester produces different results (or interprets the result differently) compared to when they are run as a stand-alone compiled exe
- Travis running tester produces a different result to tester running on my PC (for tisopr.nim)

**Comment by dom96** [comment]
@jlp765 that sounds bad, could you create a separate issue for this? Thanks.
